packages/swapper/src/swappers/cow/utils/helpers/helpers.test.ts

```diff
@@ -0,0 +1,99 @@
+import { AssetService } from '@shapeshiftoss/asset-service'
+
+import { FOX, WBTC } from '../../../utils/test-data/assets'
+import { CowSwapperDeps } from '../../CowSwapper'
+import { cowService } from '../cowService'
+import { getUsdRate } from '../helpers/helpers'
+
+jest.mock('../cowService')
+jest.mock('@shapeshiftoss/asset-service')
+
+// @ts-ignore
```

Comment:
do this properly please

```
jest.mock('@shapeshiftoss/asset-service')
const mockAssetService = jest.mocked('assetService', true)
mockAssetService.mockImplementation(...
```

Reply:
Removed asset-service dependency so this line does not exist anymore